streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -540,7 +542,7 @@ public void shouldHandleRemovedTasksFromStateUpdater() {
             "taskManagerTest",
             activeTaskCreator,
             standbyTaskCreator,
-            topologyMetadata,
+            new Tasks(new LogContext()), topologyMetadata,
```

Comment:
nit: new line

Reply:
I will add the new line in one of the following PRs.